docs/tmp.js.html

```diff
@@ -132,17 +132,18 @@ <h1 class="page-title">Source: tmp.js</h1>
 
   _assertAndSanitizeOptions(opts);
 
+  let name;
   let tries = opts.tries;
   do {
-    const name = _generateTmpName(opts);
+    name = _generateTmpName(opts);
     try {
       fs.statSync(name);
     } catch (e) {
       return name;
     }
   } while (tries-- > 0);
 
-  throw new Error('Could not get a unique tmp filename, max tries reached');
+  throw new Error('Could not get a unique tmp filename, max tries reached ' + name);
```

Comment:
Why this change was necessary? I believe it is not too useful to return a random name in the error message.

Reply:
ups, temporary change for debugging purposes... will undo